### PR TITLE
food_factAPI

### PR DESCRIPTION
**Pull request by sasukepashapoor**

### CODE REVIEW

File: crowd-server/crowd_server/apps/food_fact_API/apps.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+from django.apps import AppConfig
+
```
Comment (hr-samsami) [inline crowd-server/crowd_server/apps/food_fact_API/apps.py:2]: Code missing: add config class like the bellow
<code>
 class ImageLabelConfig(AppConfig):  
 default_auto_field ='django.db.models.BigAutoField'
 name =crowd_server.apps.image_label'
</code>

**Review by hr-samsami — CHANGES_REQUESTED** [review]
1. add your app name to INSTALLED_APPS in crowd_server/settings/base.py
2. add the path of your API to urlpatterns in crowd_server/urls.py

**Review by hr-samsami — APPROVED** [review]

File: crowd-server/crowd_server/settings/base.py [hunk 2]
```diff
@@ -43,6 +43,8 @@
     'crowd_server.apps.sentiment',
     'crowd_server.apps.food_label',
     'rest_framework',
+    'crowd_server.apps.food_fact_API'
```
Comment (ketabi) [inline crowd-server/crowd_server/settings/base.py:46]: It's better to be `'crowd_server.apps.food_fact'`

File: crowd-server/crowd_server/urls.py [hunk 3]
```diff
@@ -25,4 +25,5 @@
     path('sentiment/', include('crowd_server.apps.sentiment.urls')),
     path('food_label/', include('crowd_server.apps.food_label.urls')),
     path('food_compare/', include('crowd_server.apps.food_compare.urls')),
+    path('food_compare/', include('crowd_server.apps.Food_Fact_API.urls')),
```
Comment (ketabi) [inline crowd-server/crowd_server/urls.py:28]: `     path('food_fact/', include('crowd_server.apps.food_fact.urls')),1

File: crowd-server/crowd_server/apps/food_fact_API/apps.py [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+class FoodFactConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
+    name = 'crowd_server.apps.Food_Fact_API'
```
Comment (ketabi) [inline crowd-server/crowd_server/apps/food_fact_API/apps.py:5]: change the name

**Review by ketabi — CHANGES_REQUESTED** [review]

**Review by ketabi — APPROVED** [review]
Great

**Pull request merged**